styles/styles.css

```
@@ -0,0 +1,270 @@
+@font-face{
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face{
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+body{
+    font-family: "PT Sans", sans-serif;
+    background-color: #ffffff;
+    color: #000000;
+    font-size: 18px;
+    line-height: 21px;
+    font-weight: 400;
+}
+
+/* header */
+.page-header{
+    font-size: 20px;
+    line-height: 26px;
+    background-color: #ffffff;
+    font-weight: 700;
+}
+
+.navigation-item{
+    list-style: none;
+}
+
+.navigation-link{
+    color: #000000;   
+    text-decoration: none;
+}
+/* Стиль для кнопок */
+.button{
+    background-color: #756257;
+    color: white;
+    font-size: 16px;
+    line-height: 20px;
+    text-transform: uppercase;
+}
```

У кнопок и инпутов шрифт по умолчанию не наследуется. 
А если этот класс будет на ссылке, то стоит убрать заранее подчеркивание.

вес почти везде жирный
```suggestion
.button{
    background-color: #756257;
    font-family: inherit;
    text-decoration: none;
    color: white;
    font-size: 16px;
    line-height: 20px;
    font-weight: 700;
    text-transform: uppercase;
}
```